src/rust/bitbox02/src/ui/types.rs

```diff
@@ -42,6 +42,21 @@ impl Default for Font {
     }
 }
 
+/// Survive forces T to live at least as long as lifetme 'a.
+pub struct Survive<'a, T: 'a> {
```

Comment:
Since we're going with this, why not move it to some utils mod? To use in other places.

Reply:
Done